cypress/e2e/homeworks/hw_ddt/ddt_add_user.cy.js

```diff
@@ -0,0 +1,51 @@
+import { faker } from "@faker-js/faker";
+import { LoginPage } from "../../../page-objects/pmtool/login_page";
+import newUserData from "../../../fixtures/new_user_data.json";
+import { AddUser } from "../../../page-objects/pmtool/add_user";
+import { UsersPage } from "../../../page-objects/pmtool/users_page";
+
+describe("Data Driven Tests Homework", () => {
+  beforeEach(() => {
+    new LoginPage()
+      .openPmtool()
+      .typeUsername(Cypress.env("pmtool_username"))
+      .typePassword(Cypress.env("pmtool_password"))
+      .clickLogin()
+      .clickUsers()
+      .clickAddUserButton();
+  });
+
+  newUserData.forEach((userData) => {
+    it(`DDT Add User: ${userData.description}`, () => {
+      const userName = userData.username_prefix + faker.internet.userName();
+      const accessGroup = userData.access_group;
+      const firstName = faker.person.firstName();
+      const lastName = faker.person.lastName();
+      const userEmail = faker.internet.email();
```

Comment:
Generuješ reálně vypadající e-mail, můžeš se někomu trefit do e-mailu a to pak bude znamenat prů...švih :)